### PR TITLE
Sentient mobs can slowly subtract slime stacks by shaking

### LINKED ISSUE #82746: Slime Damage over time status effect has no wear off timer leaving some nonhuman antags with constant DOT
Reporting client version: 514.1589

## Issue Summary

Be a non-human antag (xenomorph, sentient creature or space dragon)
Get attacked by a slime, latched onto, and get the slime damage over time effect.
Be completely unable to remove it, stuck with constant damage over time that adds onto burn damage and other DOT effects.
In the case of xenomorphs it would require said xenomorph to be over weeds constantly, the rest are just screwed.

## Round ID:
[227409](https://scrubby.melonmesa.com/round/227409)

## Testmerges:
- [ai controllers use cell trackers to know when to idle](https://www.github.com/tgstation/tgstation/pull/82691)
- [Makes dropdowns better](https://www.github.com/tgstation/tgstation/pull/82697)
- [Makes it EVEN EASIER to work with atom item interactions ft. "Leaf and Branch" & "Death to Chains"](https://www.github.com/tgstation/tgstation/pull/82625)

### PR DESCRIPTION
**Pull request by Jacquerel**
## About The Pull Request

Closes #82746
PR #77569 replaced the clone damage inflicted by slimes with a lingering damage over time effect which can be removed with water.
This is all well and good for humans who can spray themselves with fire extinguishers, showers, or sinks... but most other mobs are not actually capable of opearating any form of equipment which can make them wet.

This PR allows you to click on the debuff icon to shake the slime off slowly over time (15 seconds to fully remove the debuff).
This is slower than the recovery time when wet.

Dogs, Cats, and by extension, Felinids are better at removing slime because of their self-cleaning prowess (they're always licking weird stuff off themselves). This is rarely helpful if you're not a Felinid, because almost all cats and dogs have so little health that they will hit the minimum health threshold and lose the debuff anyway by the time they would be done.
In the very niche circumstance that a sentient wolf is attacked by a slime they will appreciate this bonus. 

## Why It's Good For The Game

This was a replacement for slimes inflicting clone damage (I am not sure this even did anything for most simple mobs?) but essentially turned slimes into an automatic death sentence for most forms of mob without hands.
Technically the slime debuff cannot kill you and stops if you are under 10 HP, but this is small consolation for most of these mobs which cannot heal themselves either.

You are still probably going to take between 30-60 damage during the 15 seconds of removing slime from your body.

## Changelog

:cl:
balance: Corrosive slime left behind after a slime fails to eat you can be scraped off with your hands, or shaken off in some other way, by clicking on the debuff. This is slower and less effective than washing it off using water.
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
I was expecting this to be a `*shake` sig handler

File: code/datums/status_effects/debuffs/slime/slimed.dm [hunk 1]
```diff
@@ -8,6 +8,38 @@
 	desc = "You are covered in slime and it's eating away at you! Find a way to wash it off!"
 	icon_state = "slimed"
 
+/atom/movable/screen/alert/status_effect/slimed/Click()
+	. = ..()
+	if (!.)
+		return FALSE
+	if (!can_wash())
+		return FALSE
+	INVOKE_ASYNC(src, PROC_REF(remove_slime))
```
Comment (MrMelbert) [inline code/datums/status_effects/debuffs/slime/slimed.dm:17]: Update the desc to show this is clickable

Comment (MrMelbert) [reply code/datums/status_effects/debuffs/slime/slimed.dm:17]: Or if you wanna do something fancy we can add an animation to clickable alerts, make it wiggle or something

**Comment by MrMelbert** [comment]
Give dogs a bonus to shake off speed to be authentic too

**Comment by Jacquerel** [comment]
@MrMelbert do you think that this counts as a species change i will need to ask oranges about

**Pull request merged**